src/explore-education-statistics-admin/src/pages/bau/BoundaryDataPage.tsx

```diff
@@ -0,0 +1,198 @@
+import Page from '@admin/components/Page';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import React from 'react';
+import FormattedDate from '@common/components/FormattedDate';
+import Details from '@common/components/Details';
+import Link from '@admin/components/Link';
+import { useQuery } from '@tanstack/react-query';
+import boundaryDataQueries from '@admin/queries/boundaryDataQueries';
+import classNames from 'classnames';
+import styles from './BoundaryDataPage.module.scss';
+
+const BoundaryDataPage = () => {
+  const { data: boundaryLevels = [], isLoading: isLoadingBoundaryLevels } =
+    useQuery(boundaryDataQueries.getBoundaryLevels);
+
+  return (
+    <Page
+      title="Boundary data"
+      caption="Manage Boundary data"
+      wide
+      breadcrumbs={[
+        { name: 'Platform administration', link: '/administration' },
+        { name: 'Boundary data' },
+      ]}
+    >
+      <LoadingSpinner
+        loading={isLoadingBoundaryLevels}
+        text="Loading boundary levels"
+      >
+        <Details summary="Glossary" id="glossary">
+          <h2 className="govuk-heading-lg">Boundary types</h2>
+          <dl
+            className={classNames(styles.glossarySection, styles.boundaryTypes)}
+          >
+            <dt>BFC</dt>
+            <dd>
+              Full resolution - clipped to the coastline (Mean High Water mark)
+            </dd>
+
+            <dt>BFE</dt>
+            <dd>
+              Full resolution - extent of the realm (usually this is the Mean
+              Low Water mark but, in some cases, boundaries extend beyond this
+              to include offshore islands)
+            </dd>
+
+            <dt>BGC</dt>
+            <dd>
+              Generalised (20m) - clipped to the coastline (Mean High Water
+              mark)
+            </dd>
+
+            <dt>BUC</dt>
+            <dd>
+              Ultra Generalised (500m) - clipped to the coastline (Mean High
+              Water mark)
+            </dd>
+          </dl>
+
+          <h2 className="govuk-heading-lg">Level codes</h2>
+          <dl
+            className={classNames(
+              styles.glossarySection,
+              styles.boundaryLevels,
+            )}
+          >
+            <dt>EDA</dt>
```

Comment:
Can we use `locationsLevelsMap` here rather than hardcoding these? Could add the code (`EDA` or whatever) to `locationsLevelsMap` and then just loop over them all. This way if someone adds another location type in the future, they're less likely to miss adding it here.

Reply:
Possibly, wasn't aware of that so I'll have a look how it's used elsewhere to make sure adding another property doesn't break stuff.

YAGNI principle put me off adding reusability complexity to some of this work, that's why most of the view stuff is in one file.

Reply:
I think I'm keen on it because we have had to add new location types multiple times and I think every time we've missed updating one place or another, so I think anything to reduce that burden is worthwhile.

Reply:
It looks like `locationLevelsMap` is a dictionary, rather than a list I can iterate over.

I can add codes to the existing object, but I'd still need to manually query each one by a key, rather than just adding a simple loop, so I'm not convinced on the benefit of refactoring this from hard-coded. The same applies to the boundary types (assuming I mimic the dictionary structure from location levels).

Reply:
Dictionary is just a custom type we've written to wrap up TS's `<Record>` type anyway. It's just an object - so I think we could just iterate over it like so: 
<img width="384" alt="image" src="https://github.com/dfe-analytical-services/explore-education-statistics/assets/63285990/31e54836-f0ec-4f2d-91c9-5c2b734a7933">

(We could also return that as actual JSX rather than the interpolated string, but I was just being quick to prove the point)

Reply:
Tom has a very good point with YAGNI, but my personal tiebreaker votes goes to:
<img width="451" alt="image" src="https://github.com/dfe-analytical-services/explore-education-statistics/assets/63285990/4165ce5f-a56b-4268-8e26-9eecd9fcd866">